assets/prototype/application/services/config/test/data.ts

```diff
@@ -23,8 +23,8 @@ export const mockEeJsData: JsDataProps = {
 		site_url: '',
 	},
 	default_timezone: {
-		pretty: 'ManZooria',
-		string: 'Manzooropa/ManZooria',
-		offset: +27.333,
+		pretty: 'Calcutta',
+		string: 'Asia/Calcutta',
+		offset: +5.5,
```

Comment:
You didn't like `ManZooria` or it's not parsing it correctly?

Reply:
I would have been eternally there but JS doesn't know (🤷‍♂ ) about me 😎